bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/ui/view/variant/VariantDecorator.java

```diff
@@ -14,72 +14,67 @@
 
 public class VariantDecorator implements GSubElementDecorator<Schedule, Variant, VariantAttribute> {
 
-//	private static final Logger LOGGER = Logger.getLogger(VariantDecorator.class.getName());
-	
-	public static final Icon ICON_VARIANT = SharedIcons.ICON_VARIANT;
-	public static final Icon ICON_VARIANT_WARNING = new CompositeIcon(ICON_VARIANT, SharedIcons.OVL_WARN);
-	public static final Icon ICON_VARIANT_ERROR = new CompositeIcon(ICON_VARIANT, SharedIcons.OVL_ERROR);
-	
-	public void decorate(JLabel label, Variant element, VariantAttribute subElement, Object value) {
-		
-		if (subElement == VariantAttribute.Name) {
-			
-			// Icon
-			Severity sev = element.getSeverity();
-			if (sev != null) {
-				switch (sev) {			
-				case Error:	  label.setIcon(ICON_VARIANT_ERROR);   break;				
-				case Warning: label.setIcon(ICON_VARIANT_WARNING); break;
-				default:	  label.setIcon(ICON_VARIANT); 				
-				}
-			} else {
-				label.setIcon(ICON_VARIANT); 	
-			}
-			
-			// Alignment
-			label.setHorizontalAlignment(SwingConstants.LEFT);
-			
-		} else if (subElement == VariantAttribute.Wind) {
-
-			if (value == null) value = "-";
```

Comment:
I'm not sure why it was there in the first place but the reassignment of `value` was never used